ab_tool/templates/ab_tool/experimentsDashboard.html

```diff
@@ -203,6 +203,10 @@ <h3 class="panel-title panel-title-experiment">Experiment "{{experiment.name}}"<
 
 <script type="text/javascript">
 $(document).ready(function(){
+    function is_valid_url(url) {
+        return /^(http(s)?:\/\/)?(www\.)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$/.test(url);
```

Comment:
Unit tests should be written for this, as this complex regex is difficult to manually parse. This javascript validation step will the third, following the existing HTML5 url type on the input field, and before the backend URL validation,